docs/Documentation/publish-via-cli.md

```diff
@@ -0,0 +1,27 @@
+# Publish from Command Line 
+
+![Publish Via CLI](../../assets/images/publish-via-cli.png)
+
+1. Install the @stoplight/cli package from NPM 
+
+```
+Npm i -g @stoplight/cli
+
+# The “stoplight” cli should now be available globally 
+
+Stoplight help 
+```
+
+2. Make sure you are in the root directory of the local git repository associated with this project 
+
+```
+cd directory/to/analyze 
+```
+
+3. Run the analyze command, passing in the secret token associated with this project, and the Stoplight API host. This will analyze all the design-related files in the local project directory, and send the information to Stoplight to be displayed as published docs, included in the search explorer, etc. 
+
+```
+stoplight analyze --token ************ --api-host https://stoplight.io/api 
```

Comment:
Suggest leaving off `--api-host` - it's only relevant to enterprise customers with on premise installations. We can add a section for this later?

```
stoplight analyze --token ************
```